README.md

```diff
@@ -204,6 +204,35 @@ const sixhundredtwelve = 612;
 assert(helpers.is16(sixhundredtwelve));
 ```
 
+#### ProxySocket
+A utility that can be passed to a bcoin or bcash node that you would like to run in
+the browser.
+
+```js
+import { ProxySocket } from '@bpanel/bpanel-utils';
+
+const node = new FullNode({
+      hash: true,
+      query: true,
```

Comment:
...never did figure out what `hash` and `query` parameters are for, are they deprecated?

Reply:
haha, oh yeah. Let me take those out.